lib/rubocop/cop/performance/inefficient_hash_search.rb

```diff
@@ -87,7 +90,7 @@ def autocorrect_argument(node)
         end
 
         def autocorrect_hash_expression(node)
-          node.children[0].children[0].loc.expression.source
+          node.children[0].receiver.loc.expression.source
```

Comment:
You can probably use just `Node#source`, which is an alias for `loc.expression.source`. (Internal investigation incoming. 😉)